src/mme-app/actionHandlers/serviceRequestActionHandlers.cpp

```diff
@@ -133,16 +134,24 @@ ActStatus ActionHandlers::send_ddn_ack_to_sgw(ControlBlock& cb)
 	    return ActStatus::HALT;
 	}
 
```

Comment:
Open a enhancement in MME to send DDN Ack with -ve cause if no UE context found.

Reply:
Opened EDGEPOD-421.